test/add2-reshape.cc

```diff
@@ -0,0 +1,149 @@
+// Copyright 2023 Google LLC
```

Comment:
nit: same for this binary element wise op - move to another PR?

Reply:
Same as previous comment